src/Serializer/AbstractItemNormalizer.php

```diff
@@ -144,7 +144,7 @@ public function normalize($object, $format = null, array $context = [])
      */
     public function supportsDenormalization($data, $type, $format = null)
     {
-        if (ItemNormalizer::FORMAT === $format) {
+        if (ElasticsearchItemNormalizer::FORMAT === $format) {
```

Comment:
Why this change?

Reply:
Because it's easier to read, it can be confusing if you read `ItemNormalizer`